### DISCUSSION TITLE
Blog Post for Round 20; Request for Maintainers

### DISCUSSION THREAD

**Discussion by NateBrady23** [TechEmpower/FrameworkBenchmarks#6214] [General]
Hi everyone!

Round 20 is fast approaching (again) and for the official blog post entry we'd like to include a section from maintainers about the work they've done between Round 19 and 20 to improve performance.

What we're looking for is just a brief summary, but you can include a link to your own blog post or website that includes more details.

Feel free to leave that summary in this thread, if you'd like to take part!

**Comment by NateBrady23** [comment]
Pinging some folks here in case you haven't seen our new discussions area and to see if you'd like to add anything for the official blog entry.

@benaadams @sagenschneider @matt-42 @dantti @sebastienros @FroMage @JHyeok

**Reply by sebastienros** [reply]
Stephen Toub wrote a great blog post about all the things that were improved in .NET 5, including all the work that was driven by TE. Would make sense to include it. https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-5/

**Reply by FroMage** [reply]
Thanks for the ping, but I didn't find where to see the current nightly/pending results. I'd like to verify them before potentially adding anything to the blog :)

**Reply by FroMage** [reply]
https://tfb-status.techempower.com/ reports an error to me: `502 Bad Gateway`

**Reply by sagenschneider** [reply]
Save you waiting and let Nate have some sleep, temp solution available - https://github.com/TechEmpower/FrameworkBenchmarks/issues/5709#issuecomment-754576921

**Comment by benaadams** [comment]
I can't claim at the moment to have some kind of 2020 hindsight about Round 20 in 2020 🥁

**Comment by matt-42** [comment]
Thanks for the ping @nbrady-techempower. For lithium, the only major optimisation is the implementation of batch mode for postgresql.

**Comment by dantti** [comment]
Thanks @nbrady-techempower, on the Cutelyst side, I'm actually waiting for the merge of PR to see if plain text performance return to where it was.
Besides that I've created the [ASql](https://github.com/cutelyst/asql) library to talk to PostgreSQL in async mode which improved performance a bit.
I have done some test enabling LTO but I'd like to enable it after one run of my last PR to be able to compare the results...

**Comment by sagenschneider** [comment]
Thanks, was not aware and very happy to contribute :+1:    OfficeFloor summary for round 20 is as follows

OfficeFloor has introduced support for asynchronous programming in it's [inversion of coupling control](https://sagenschneider.blogspot.com/2019/02/inversion-of-coupling-control.html).  This has enabled using reactive libraries such as R2DBC.  More information can be found in [OfficeFloor's blog post about round 20](https://sagenschneider.blogspot.com/2021/01/performance-improvements.html).

**Comment by bhauer** [comment]
Thanks everyone. I aim to post Round 20 on Monday 2021-02-08.

**Comment by bhauer** [comment]
Round 20 is live!  You can read the [blog entry with the above updates](https://www.techempower.com/blog/2021/02/08/framework-benchmarks-round-20/) or [check out the results](https://www.techempower.com/benchmarks/#section=data-r20).  Note that the results are probably already well known to contributors since they're from continuous runs that have been on _tfb-status_ for a few weeks.

Thanks again, everyone!
